src/pages/_proxied-dot-io/waypoint/index.tsx

```diff
@@ -119,133 +69,149 @@ function HomePage(): JSX.Element {
-							width: 65,
-							height: 42,
-						},
-						text: <>Infinitely extensible with Waypoint&#160;plugins</>,
-					},
-				]}
-				heading="Ready to get started?"
-				content="Start by following a tutorial to deploy a simple application with Waypoint or learn about how the project works by exploring the documentation."
-				links={[
-					{
-						text: 'Get Started',
-						url: 'https://learn.hashicorp.com/collections/waypoint/get-started-kubernetes',
-					},
-					{ text: 'Explore Documentation', url: '/docs' },
-				]}
-			/>
-		</div>
+			</div>
+		</>
 	)
 }
 
-export function getStaticProps() {
-	// This function intentionally left blank to allow Next to use SSG.
-	return { props: {} }
+export async function getStaticProps() {
+	const query = proxiedRivetClient('waypoint')
+	const { waypointHomepageCurrent } = await query({
+		query: homepageQuery,
+	})
+
+	const {
+		seo,
+		heroHeading,
+		heroDescription,
+		heroCta,
+		introLeftColumnHeading,
+		introLeftColumnDescription,
+		introLeftColumnFeatures,
+		introRightColumnHeading,
+		introRightColumnDescription,
+		introRightColumnFeatures,
+		configureYourAppHeading,
+		configureYourAppCodeNote,
+		configureYourAppFeatures,
+		buildAndDeployHeading,
+		buildAndDeployFeatures,
+		monitorYourAppHeading,
+		monitorYourAppFeatures,
+		monitorAppHealthHeading,
+		monitorAppHealthDescription,
+		extendPluginsHeading,
+		extendPluginsDescription,
+		extendPluginsFeatures,
+		callToActionHeading,
+		callToActionContent,
+		callToActionFeatures,
+		callToActionLinks,
+	} = waypointHomepageCurrent
+
+	const pageData = {
+		seo,
+		hero: {
+			heading: heroHeading,
+			description: heroDescription,
+			cta: heroCta[0],
+		},
+		intro: {
+			leftColumn: {
+				heading: introLeftColumnHeading,
+				description: introLeftColumnDescription,
+				features: formatFeatures(introLeftColumnFeatures),
+			},
+			rightColumn: {
+				heading: introRightColumnHeading,
+				description: introRightColumnDescription,
+				features: formatFeatures(introRightColumnFeatures),
+			},
+		},
+		configureYourAppSection: {
+			heading: configureYourAppHeading,
+			codeNote: configureYourAppCodeNote,
+			features: formatFeatures(configureYourAppFeatures),
+		},
+		buildAndDeploySection: {
+			heading: buildAndDeployHeading,
+			features: formatFeatures(buildAndDeployFeatures),
+		},
+		monitorYourAppSection: {
+			heading: monitorYourAppHeading,
+			features: formatFeatures(monitorYourAppFeatures),
+		},
+		monitorAppHealthSection: {
+			heading: monitorAppHealthHeading,
+			description: monitorAppHealthDescription,
+		},
+		extendPluginsSection: {
+			heading: extendPluginsHeading,
+			description: extendPluginsDescription,
+			features: formatFeatures(extendPluginsFeatures),
+		},
+		callToActionSection: {
+			heading: callToActionHeading,
+			content: callToActionContent,
+			features: callToActionFeatures,
+			links: callToActionLinks,
+		},
+	}
+
+	return {
+		props: {
+			data: pageData,
+		},
```

Comment:
Probably want to add revalidate here to allow previewing changes https://github.com/hashicorp/dev-portal/blob/main/src/pages/_proxied-dot-io/vault/index.tsx#L168

Reply:
Ah! Of course, added 👍